src/components/tile-implementations/earthquake/useEarthquakeApi.test.ts

```
@@ -0,0 +1,111 @@
+};
+
+global.fetch = vi.fn();
+
+describe('useEarthquakeApi', () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+    registerEarthquakeDataMapper();
+  });
+
+  it('fetches and maps USGS earthquake data successfully', async () => {
+    (globalThis.fetch as unknown as { mockResolvedValueOnce: Function }).mockResolvedValueOnce({
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)